### DISCUSSION TITLE
Link static library of imagemagick and all its dependencies in xcode project

### DISCUSSION THREAD

**Discussion by PS1814** [ImageMagick/ImageMagick#6599] [Help]
### ImageMagick version

7

### Operating system

MacOS

### Operating system, version and so on

MacOS Ventura

### Description

Can you please guide me how can I get static libraries of ImageMagick and its dependencies. And how can I link them in my xcode project?

Thanks

### Steps to Reproduce

Can you please guide me how can I get static libraries of ImageMagick and its dependencies. And how can I link them in my xcode project?

Thanks

**Comment by dlemstra** [comment]
Please use discussions instead of issues for these kind of questions.

**Reply by PS1814** [reply]
Sure, will take care of it next time.
Can you please help me with this? It would be of great help.
